typescript/processors/readTypeScriptModules.js

```diff
@@ -0,0 +1,451 @@
+var glob = require('glob');
+var path = require('canonical-path');
+var _ = require('lodash');
+var ts = require('typescript');
+
+module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo, untouchedNamespaces,
+                                                getExportDocType, getContent, createDocMessage, log) {
+
```

Comment:
I was able to fake out the `readFilesProcessor` issue by including it and setting a `sourceFiles` object that would resolve to no matching files. However you'll still run into problems with that configuration as `readTypeScriptModules.sourceFiles` only works with an array of plain glob strings, not objects with include and exclude patterns.